iac/compositions/cat-full/ssm_params.tf

```diff
@@ -0,0 +1,44 @@
+locals {
```

Comment:
Super semantic point here...

In general I try to avoid naming a Terraform file for the types of resource it contains. (The exception here is if there is typically only _one_ of the things in any composition, then I'll call the file that, e.g. `ecs_cluster.tf`)

The reason I avoid this in general is that when you do this, it suggests that ALL resources of type X will be in that file whereas in fact it's often desireablt to place SOME resources of type X according to their business function in the same tf file as other resource types.

So while this file is indeed all about SSM params it might not be about ALL the SSM params. How about `parameter_sourcing.tf` or something along those lines?

_[btw I fully admit I myself am not always in line with this recommendation]_